### PR TITLE
FUSETOOLS2-137 - changelog filtered completion list properties file

### PR DESCRIPTION
**Pull request by apupier**
requires https://github.com/camel-tooling/camel-language-server/pull/347

# Pull Request informations

## Rebase & Merge default requirements

1. Green build for master branch
2. Wait 24 hours after PR creation
3. Green job for PR
4. Approved PR

## PR labels default process

- READY_FOR_REVIEW  &rightarrow;  REVIEW_DONE  &rightarrow;  READY_FOR_MERGE

## Tests

- [ ] Are there **Unit tests**?
- [ ] Are there **Integration tests**?
- [ ] Do we need a new **UI test**?

## PR workflow progress

1. [ ] Tagged with relevant **PR labels**
2. [ ] Green **job for PR**
3. [ ] PR was created more than **24 hours ago** or **All committers approved** it
4. [ ] Green **master** branch build

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Completion for possible enum values of a Camel component property in properties file
 - In Properties file, the default values are automatically added when auto-completing Camel component properties
+- In Properties file, provide filtered completion when in middle of a component id, component property or value 5Previously, completion was available only right after the dot or equal separators.
```
Comment (lhein) [inline Changelog.md]: typo in here "5Previously"

File: Changelog.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 - In properties file, completion for possible enum values and booleans of a Camel component property 
 - In Properties file, the default values are automatically added when auto-completing Camel component properties
+- In Properties file, provide filtered completion when in middle of a component id, component property or value. Previously, completion was available only right after the dot or equal separators.
```
Comment (lhein) [inline Changelog.md]: btw. now that I read it again. Why do we have properties sometimes with capital P and sometimes with lower case p?

Comment (apupier) [reply Changelog.md]: because I wasn't coherent :-(
I will update it in this PR

Comment (apupier) [reply Changelog.md]: done

**Pull request merged**